### DISCUSSION TITLE
Primeflex over tailwind

### DISCUSSION THREAD

**Discussion by sasafister** [primefaces/community#1955] [PrimeVue Blocks] [answered]
I'm using PrimeBlocks and Vue2 with tailwind. The problem is that I have everything setup with tailwind, and I want to use Block (obvious I need PrimeFlex for that). If I import Primeflex in app.js, then it takes over tailwind for some classes, and messes UI completely. Any solution?

**Comment by tugcekucukoglu** [comment, marked as answer]
We recommend to choose PrimeFlex or Tailwind. Officially v4 uses Tailwind. Using PrimeCLT you could convert PrimeFlex project to Tailwind. For more information: https://v4.primevue.org/primeclt/

**Comment by almirhodzic** [comment]
> I'm using PrimeBlocks and Vue2 with tailwind. The problem is that I have everything setup with tailwind, and I want to use Block (obvious I need PrimeFlex for that). If I import Primeflex in app.js, then it takes over tailwind for some classes, and messes UI completely. Any solution?

@sasafister: Did the recommended solution works for you?
